Dockerfile

```diff
@@ -5,7 +5,6 @@ WORKDIR /work
 
 ADD . /work
 
-ENV SCONS_LIB_DIR="/usr/local/lib/python3.7/dist-packages/scons"
```

Comment:
This may not have worked because we have Python3.8. So drop it.

Reply:
I think this was a specific issue with Ubuntu 18.04 or with the Python 3.7 or something.